docs/index.md

```diff
@@ -18,62 +19,40 @@ hide_title: true
   <a href="https://twitter.com/webmixedreality" target="_blank">Twitter</a>
 </div>
 
-<h4 align="center">🚧 This new site is under active development and so many things are still missing. [Contributions are welcome!](https://github.com/webaverse/docs) 🧰</h4>
 <h3 align="center">⚠️ M3 is hosting a hackathon for Webaverse creations! [Find more information on participating here!](m3-hackathon.md) 🔍</h3>
 
-## ❓ What Are Augs?
+## ❓ What is the Webaverse?
 
-Augs are creative spatial apps that run everywhere. Unlike traditional 2D applications, augs exist in three dimensions and can be experienced with a desktop browser or any SteamVR compatible headset. Augs are created and owned by users like you! Getting started with augs today is easy, keep reading below to get started, or see the more page [`What Is Webaverse?`](about.md) for a more in-depth explanation.
+<a href="https://webaverse.com/" target="_blank" rel="noopener noreferrer">The Webaverse</a> is a blanket term used to describe a set of technologies that enable the use of spatial applications and objects powered by WebXR. These spatial applications are called "augs".
```

Comment:
I think we went with "xrpk"s, since that is what folks ended up calling them. That change would probably need to be applied across the entire doc.